### DISCUSSION TITLE
Project status renaming

### DISCUSSION THREAD

**Discussion by kranta-vocal** [community/community#142468] [New to GitHub] [answered]
Hello, I am working on projects and I would like to add custom status to my projects.

Could someone point me to how to edit them? Or if it is not possible please let me know 

Thank you
![image](https://github.com/user-attachments/assets/e2ea7ec3-3ed4-45ef-be0d-952c84d86610)

**Comment by buritbest123** [comment]
In GitHub, specifically in GitHub Projects, it is not currently possible to add custom statuses directly like you might in more traditional project management tools (e.g., Jira). GitHub provides pre-defined status options such as "On track," "At risk," and "Off track" within their Projects tool, as shown in your screenshot.

If you'd like more flexibility in defining project statuses, here are a few alternatives you can consider:

1. **Use Labels:**
   - You can create custom labels for your issues or pull requests (e.g., "In Review," "Needs Attention").
   - Navigate to the **Issues** or **Pull Requests** tab.
   - Select **Labels** from the sidebar.
   - Click **New label** to create a custom status label.
   - Use these labels to mimic custom statuses.

2. **GitHub Actions:**
   - Automate the assignment of statuses based on certain conditions. For example, a GitHub Action could be set up to apply a specific label based on the number of days an issue has been open, the type of issue, or other criteria.

3. **Custom Columns in GitHub Projects:**
   - You can add custom columns in your project board to represent different statuses.
   - For example, you could create columns for custom stages like "Under Review," "Blocked," or "Needs Revision."
   
4. **Third-Party Integrations:**
   - Consider using third-party integrations such as ZenHub or Clubhouse that extend GitHub's project management capabilities, including custom workflows and statuses.

😊

**Comment by kranta-vocal** [comment]
Thank you so much for the reply is there a pla to make it customisable?

**Reply by EliasDeHondt** [reply, marked as answer]
**Hi @kranta-vocal !!!**

As of now, GitHub hasn’t officially announced plans for fully customizable statuses in GitHub Projects. However, feature requests for greater flexibility, like custom statuses, are common, and GitHub does consider community feedback when planning updates. It’s possible we may see this functionality in future releases, so I’d recommend keeping an eye on GitHub’s changelog or their discussions board for any news on this front.

If you found this information useful, please feel free to mark it as the correct answer. 😊
